### ISSUE TITLE
Global config for slug builder

### ISSUE THREAD

**Issue by johnnyshields** [digitalplaywright/mongoid-slug#166]
I'd like to raise a PR for a global config for alternate slug generation (currently it must be configured in each model.) Something like the following in an initializer:

```
Mongoid::Slug.config do |c|
  c.slug_builder = ->{ my_proc }
end
```

In most cases, I would think users (like myself) would want the slug logic to be consistent across their app. In my case, I need to convert Japanese to alphabet and current stringex logic doesn't do a good job (it assumes the characters as Chinese, not Japanese)

Please confirm you'd accept this PR if raised?

**Comment by anujaware** [comment]
@johnnyshields: I added above feature. Thanks for this discussion. It helped me to implement the solution. I hope this will help.

**Comment by anujaware** [comment]
@dblock I think we can close this issue.

**Comment by dblock** [comment]
Closed via https://github.com/digitalplaywright/mongoid-slug/pull/196.

**Comment by dblock** [comment]
FYI was released in 5.3.0
